onebusaway-android/src/main/java/com/joulespersecond/seattlebusbot/Application.java

```diff
@@ -45,6 +49,18 @@
 
     private static Application mApp;
 
+    private static final String PROPERTY_ID = "UA-57764191-1";
```

Comment:
`PROPERTY_ID` constant should be removed here, since its defined in app-tracker.xml and global_tracker.xml